### PR TITLE
Carts on sites experience slot

### PR DESCRIPTION
**Pull request by geoffreymm**
Add docs to the feature introduce on this [PR](https://github.com/easolhq/easol/pull/9024) that allow to book a specific slot.

### CODE REVIEW

**Comment by geoffreymm** [comment]
[Documentation for ExperienceDateDrop](https://trello.com/c/N0rewtq7/147-documentation-for-experiencedatedrop)

File: docs/reference/tags/form_tags/create_line_item/create_line_item.md [hunk 1]
```diff
@@ -7,8 +7,9 @@ grand_parent: Reference
 
 # Create line item
 
-The `create_line_item` tag renders a form which acts as a wrapper for a single item, an item can be a [variant]({% link docs/reference/objects/product/variant/index.md %}) or an [extra]({% link docs/reference/objects/product/extra.md %}).
-Extra HTML input tags can be used to add item quantity, modifiers, adult count or start and end date.
+The `create_line_item` tag renders a form which acts as a wrapper for a single item, an item can be a [variant]({% link docs/reference/objects/product/variant/index.md %}) or an [extra]({% link docs/reference/objects/product/extra.md %}).<br>
```
Comment (eballantine) [inline docs/reference/tags/form_tags/create_line_item/create_line_item.md:10]: pet peeve nit: the break tag is meant for addresses & poems, you can add a blank line to start a new paragraph!

**Review by eballantine — APPROVED** [review]
Lovely documentation, excellent punctuation ✨

File: docs/reference/tags/form_tags/create_line_item/create_line_item.md (hunk 1 above)
Comment (geoffreymm) [reply docs/reference/tags/form_tags/create_line_item/create_line_item.md:10]: Adding a blank line is literally adding a blank line and not a paragraph 😞

**Pull request merged**